src/strategies/views/climate-view.test.ts

```
@@ -0,0 +1,16 @@
+import { describe, expect, test, vi } from 'vitest';
+
+vi.mock('@smolpack/hasskit', () => ({
+  Home: class {
+    constructor(_hass: unknown) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)